packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html

```diff
@@ -288,11 +287,13 @@ <h4 class="oui-box__heading" data-translate="hosting_cdn_shared_option_prewarm_t
                 <div class="mt-3">
                     <p class="mb-0"><strong data-translate="hosting_cdn_shared_option_prewarm_quota"></strong></p>
                     <p data-ng-bind="$ctrl.getPrewarmQuotaUsage()"></p>
-                    <progress
-                        class="w-25 oui-progress oui-progress_info"
-                        value="prewarm.extra.usage"
-                        max="prewarm.extra.quota">
-                    </progress>
+                    <div class="oui-input-group oui-input-group_l">
+                        <progress
```

Comment:
**thought:** Not sure that wrapping `progress` element inside a `oui-input-group` make sense.

**question:** Couldn't be replaced by [`<oui-progress>`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-progress-webcomponent--default)?

Reply:
Yep, i completed with my current fixes.